transforms/add-prop/README.md

```diff
@@ -0,0 +1,58 @@
+# add-prop
+
+This universal codemod provides possibility to add any prop to any component.
+
+```diff
+- <Component />
++ <Component newProp="info" />
+
+```
+## Usage
+
+`npx carbon-codemod add-prop <target> <component-import-path> <prop> <value>`
+
+### Examples
+
+### String
+`npx carbon-codemod add-prop src carbon-react/lib/components/button ml "16px"`
+
+```js
+import Button from "carbon-react/lib/components/button";
+```
+```diff
+- <Button>Button</Button>
++ <Button ml="16px">Button</Button>
+```
+
+### Number
+`npx carbon-codemod add-prop src carbon-react/lib/components/button ml 2`
+
+```js
+import Button from "carbon-react/lib/components/button";
```

Comment:
You don't really need to have the import on every example, if you look at the other readme's they don't so let's keep them consistent 👍🏻

Reply:
Cheers for spotting this. I'll get the import paths removed from the README.